projects/ion/src/lib/dropdown/dropdown.component.spec.ts

```diff
@@ -63,9 +63,11 @@ describe('IonDropdownComponent', () => {
   it('should change icon to close when mouse enter in option selected', async () => {
     const elementToHover = document.getElementById('option-0');
     fireEvent.click(elementToHover);
-    fireEvent.mouseEnter(elementToHover);
-    expect(screen.queryAllByTestId('ion-check-selected')).toHaveLength(0);
-    expect(screen.queryAllByTestId('ion-close-selected')).toHaveLength(1);
+    setTimeout(() => {
```

Comment:
setTimeout doesn't work correctly on test assertions. To check if this one is alright, try to change the lenghts to 100 and 101, for example, and see if the assertion fails.

Reply:
the test fails without the setTimout

Reply:
It's a false positive, setTimeout doesn't work on test assertions. Try to do a `expect(2).toEqual(3)` inside that setTimeout and it will pass, even when it's clearly incorrect. 😄

Reply:
I didn't know that lol, I'll try to fix this test today =D